changelog

```diff
@@ -1,3 +1,242 @@
+    * Added a "Find" button to the map. (@Terin)
+    * The min and max zoom levels on the map can now be customized in the interface file. (@thebigh2014)
+    * The vsync setting can now be changed in the preferences. (@tehhowch)
+    * Text truncating is now used more widely for shortening long text. (@oo13, @tehhowch)
+    * A warning dialog will appear when attempting to change the vsync setting fails. (@tatami4)
+    * The main view, map, hiring, and trading panels are now capable of using the numpad + and - keys for zooming in/out, hiring/firing crew, or buying/selling goods. (@tatami4)
+  * Under the hood:
+    * Carried ships are now identified by their category instead of a boolean. (@petervdmeer)
+    * The game will now report more errors in conversations, including incorrect indentation. (@NomadicVolcano)
+    * The --ships flag now outputs the number of guns and turrets on a ship. (@fcfort)
+    * The systems that were defined in the Pug file have been moved to the map file. (@Zitchas)
+    * Support test case execution, locally and in CI/CD. (@petervdmeer)
+    * Copy constructor and assignment operators for File now close own file, preventing a memory leak. (@tehhowch)
+    * A single bind + unbind is used when drawing mission system rings instead of one for each ring. (@tehhowch)
+    * Commands can now be created from datafile text. (@petervdmeer)
+    * Added CLion IDE files to the gitignore. (@Ririshi)
+    * Various sections of a save file, including the outfits on a ship and visited systems and planets, are now saved in alphabetical order instead of the save order being random. (@petervdmeer, @tehhowch)
+    * Data files encoded by UTF-8 are now supported. (@flaviojs, @oo13)
+    * Fixed a potential compiler error for older compilers. (@thebigh2014)
+    * The Code::Blocks project file now uses default MinGW paths. (@Terin)
+    * Added nullptr checks where they were missing. (@tehhowch)
+    * Updated the copyright file with a "catch-all" line for contributors. (@oo13)
+    * Pre/post-fix operators now function as expected on dates. (@tehhowch)
+    * Added a missing virtual destructor for ItemInfoDisplay. (@tehhowch)
+    * Prevented a race condition scenario when loading audio. (@Rakete1111)
+    * Created a unit testing framework. (@tehhowch)
+    * Added an initial framework for integration testing. (@petervdmeer, @tehhowch)
+    * The game will now fallback onto using shaders for swizzling if the player's hardware does not have the swizzle extension. (@janisozaur, @thomasballinger)
+    * References to undefined data are now preserved in the save file, preventing crashes or lost data caused by removing plugins. (@tehhowch)
+    * The game will now warn about having a low monitor frame rate, which causes the game to run at a reduced frame rate. (@tehhowch)
+    * A bunch of CI/CD updates. (@MCOfficer, @petervdmeer, @tehhowch)
```

Comment:
```suggestion
  * CI/CD and Development Environment
    * Make (test-)executables available (for Linux, MacOS and Windows) for every in-progress PR. (@MCOfficer)
    * Update CI/CD environment to the latest GitHub actions and Linux runtime environments. (@MCOfficer)
    * Improved CI/CD stability by optimizing the package dependencies and repository servers. (@MCOfficer)
    * Organize CI/CD jobs into build and test runs and improve input handling. (@MCOfficer)
    * A number of smaller CI/CD updates. (@MCOfficer, @petervdmeer, @tehhowch)
```
I think MCOs contributions are a bit more than just a bunch of CI/CD updates. Some examples in the suggested change here. (Not sure if those were all in this release though, but based on the dates I expect they are.)

Reply:
Judging by commit date, CI reorganization and repo shenanigans were part of 0.9.12.

```suggestion
  * CI/CD and Development Environment
    * Make (test-)executables available (for Linux, MacOS and Windows) for every in-progress PR. (@MCOfficer)
    * Update CI/CD environment to the latest GitHub actions and Linux runtime environments. (@MCOfficer)
    * A number of smaller CI/CD updates. (@MCOfficer, @petervdmeer, @tehhowch)
```